db/migrate/20230125104957_create_coupon_plans.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateCouponPlans < ActiveRecord::Migration[7.0]
+  def change
+    create_table :coupon_plans, id: :uuid do |t|
+      t.references :coupon, type: :uuid, index: true, null: false, foreign_key: true
+      t.references :plan, type: :uuid, index: true, null: false, foreign_key: true
+
+      t.timestamps
+    end
+
+    add_column :coupons, :limited_plans, :boolean, default: false, null: false
```

Comment:
Do we want to merge this one as well?

Reply:
I would say yes, it has no affect and it's not exposed. Also it's false by default and that would be default behaviour